tests/test_logic/test_prop_access.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+
+import ast
+
+from wemake_python_styleguide.logic import prop_access
+
+
+def test_accesses():
```

Comment:
Sorry, but this code still does not look right. I will explain in one of the next comments.

Reply:
Done! Please, remove this test and `accesses` function.